### ISSUE TITLE
Spurious complaints about missing 'src' folder

### ISSUE THREAD

**Issue by aubade** [bruno-medeiros/DDT#105]
DUB works fine with projects that don't use a folder called src, so long as a folder is specified--and DDT is fully capable of building and running such projects. However, you'll get an error popping up in the errors tab at startup complaining about the missing src folder. 

It might be better to check and see if explicit source folders are defined before issuing the error.

**Comment by bruno-medeiros** [comment]
Can you post the error here?

**Comment by aubade** [comment]
Bah, it was being very consistent about it for a while but I'm not able to get the error now. Next time it pops up I'll try and grab the text/screenshots of it.

**Comment by bruno-medeiros** [comment]
Hum, it could have been an issue due to the project not being recreated with the new DDT 0.11.0 version. Just let me know if it happens again.

**Comment by aubade** [comment]
AH hah! Had it happen again--The other missing source files are because of the way I had to specify folders that are in the main project folder from within a subpackage. (i.e. they're in ICE/Libraries and included from ICE/Engine) and the rest is related to updating my project to work with 2.067; i just didn't want to mess with anything while I got the screenshot.

I can't get it to give me any more details on the error; double clicking does nothing and the "go to" option is greyed out.

![screenshot 2015-03-25 10 31 37](https://cloud.githubusercontent.com/assets/436747/6830930/74e9c82c-d2da-11e4-90d8-07a469c12b2c.png)
